packages/web-react/src/types/grid.ts

```diff
@@ -0,0 +1,28 @@
+import { ElementType, JSXElementConstructor } from 'react';
+import { ChildrenProps, StyleProps } from './shared';
+
+export enum GridVariant {
+  XS = 1,
+  SM = 2,
+  MD = 3,
+  LG = 4,
+  XL = 6,
+  XXL = 12,
```

Comment:
As discussed on Slack, please list just the numbers here. Introducing variant aliases is something that should be coined into our DS vocabulary so we would have to discuss with designers too. We can change it if there is a confusion on consumers' side. Is it OK?

Reply:
I have already changed it.